### ISSUE TITLE
Greater contrast between text and background in the backend

### ISSUE THREAD

**Issue by krosemichael** [ManifoldScholar/manifold#1947]
## Feature Description

Greater contrast between text and background in the backend

## *Why* is this feature important? *Who* does it help?

This is an accessibility issue

## User Stories

I am a Manifold user with a vision impairment; I am having trouble distinguishing the text from the background in the backend. I would benefit from greater contrast that would make it easier to see. 

## Design Notes

Making the text white instead of grey would solve the issue.

**Comment by zdavis** [comment]
Thanks, Krystyna!

This is going to be quite a bit more complicated than just changing all background text colors to white. We have multiple shades of gray in the background, which are used to express information hierarchy in various ways. Changing everything to white would demolish these hierarchies, and would actually work against both accessibility and usability—instead, we would need to carefully adjust each gray to meet WCAG requirements.

We've done a study on what would be entailed in addressing this and recorded the results in #1126. If we want to reopen #1126, that's fine—whether or not we open it, I'd prefer to pursue this in #1126, since the results of our research on this issue is already recorded there. We have done corresponding studies for the reader (#1125) and the frontend (#1124).

best,
Zach

**Comment by zdavis** [comment]
Also, sorry not to have mentioned #1126 sooner—it's sometimes hard to keep all this stuff straight, and I just forgot that we did that those studies.

**Comment by krosemichael** [comment]
Hi Zach,

Thank you and sorry I missed the previous ticket on this topic! Would pursuing this on #1126 mean that this discussion should be on that ticket? 

All best,
Krysia

**Comment by zdavis** [comment]
No worries. Yeah, I think that would be best. I'll close this and we can discuss further on #1126
